test/document.spec.js

```diff
@@ -546,22 +547,53 @@ describe('Document', () => {
 
   describe('Lock Status', () => {
     describe('#lock', () => {
-      it('should lock the document', () => (
+      const lockDocument = () => (
         repository.fetch(FILE_TEST_PATH)
           .then((doc) => doc.lock())
           .then((doc) => doc.fetchLockStatus())
           .then((status) => {
             expect(status.lockOwner).to.exist();
             expect(status.lockCreated).to.exist();
           })
-      ));
+      );
+
+      it('should lock the document', lockDocument);
 
-      /* eslint prefer-arrow-callback: 0 */
       it('should throw an error when locking a document already locked', function f(done) {
+        if (nuxeo.serverVersion.gt(LTS_2019)) {
+          this.skip();
+        }
+
         repository.fetch(FILE_TEST_PATH)
           .then((doc) => doc.lock())
           .then(() => done('Must not resolved when locking a document already locked'), () => done());
       });
+
+      it('should not fail on a locked document if the user is the lock owner', function f(done) {
+        if (nuxeo.serverVersion.lte(LTS_2019)) {
```

Comment:
Shouldn't it be 
```javascript
if (nuxeo.serverVersion.lt(LTS_2019)) {
```
?

As https://jira.nuxeo.com/browse/NXP-24359 was done only for `11.1`. Note that tests are failing on Nuxeo 10.10: https://qa2.nuxeo.org/jenkins/job/Client/job/nuxeo-js-client-vs-10.10/job/PR-89/17/ (see the status on the PR).

Reply:
My bad, seems not linked to that... :(